fs/_version.py

```diff
@@ -1,3 +1,3 @@
 """Version, used in module and setup.py.
 """
-__version__ = "2.4.11"
+__version__ = "2.4.12"
```

Comment:
Can you remove the bump for the version number ? We only update it in the commit where we publish a new release.

Reply:
Fixed.